src/setup-java.ts

```diff
@@ -15,16 +22,32 @@ async function run() {
 
     await installer.getJava(version, arch, jdkFile, javaPackage);
 
-    const matchersPath = path.join(__dirname, '..', '.github');
+    const matchersPath = path.join(__dirname, '..', '..', '.github');
     console.log(`##[add-matcher]${path.join(matchersPath, 'java.json')}`);
 
-    const id = core.getInput('server-id', {required: false}) || undefined;
+    const id = core.getInput('server-id', {required: false}) || DEFAULT_ID;
     const username =
-      core.getInput('server-username', {required: false}) || undefined;
+      core.getInput('server-username', {required: false}) || DEFAULT_USERNAME;
```

Comment:
Given that there are new inputs, this can also go into a constants file. Something like this: https://github.com/actions/download-artifact/blob/main/src/constants.ts